acl_loader/main.py

```diff
@@ -506,30 +590,32 @@ def show_table(self, table_name):
         :param table_name: Optional. ACL table name. Filter tables by specified name.
         :return:
         """
-        header = ("Name", "Type", "Binding", "Description")
+        header = ("Name", "Type", "Binding", "Description", "Stage")
 
         data = []
         for key, val in self.get_tables_db_info().iteritems():
             if table_name and key != table_name:
                 continue
 
+            stage = val.get("stage", Stage.INGRESS).lower()
+
             if val["type"] == AclLoader.ACL_TABLE_TYPE_CTRLPLANE:
                 services = natsorted(val["services"])
-                data.append([key, val["type"], services[0], val["policy_desc"]])
+                data.append([key, val["type"], services[0], val["policy_desc"], stage])
```

Comment:
do we need to specify the `ingress` stage for control plane ACLs?

Reply:
control plane ACLs are in INPUT chain. Does it make sense to mark them as ingress in this case?